src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.tsx

```diff
@@ -54,9 +54,8 @@ const LXDClusterDetails = (): JSX.Element => {
   const fetchedVmCluster = !gettingVmCluster && vmCluster;
 
   const loaded = clustersLoaded || fetchedVmCluster;
-  const [headerContent, setHeaderContent] = useState<KVMHeaderContent | null>(
-    null
-  );
+  const [sidePanelContent, setSidePanelContent] =
+    useState<KVMHeaderContent | null>(null);
```

Comment:
Should KVMHeaderContent be KVMSidePanelContent?

Reply:
Yes indeed, good spot.